### PR TITLE
/version endpoints: display base tables and branch table

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: server/templates/version.html [hunk 1]
```diff
@@ -58,5 +70,4 @@
       </tbody>
     </table>
   </div>
-  <script src={{url_for('static', filename='translator.js', t=config['VERSION'])}}></script>
```
Comment (pradh) [inline server/templates/version.html:61]: is this relevant to this change?

**Review by pradh — APPROVED** [review]
Does this change require that mixer + website are updated together?

File: server/templates/version.html (hunk 1 above)
Comment (shifucun) [reply server/templates/version.html:61]: This was an old bug. The version page should not include translator js

**Comment by shifucun** [comment]
> Does this change require that mixer + website are updated together?

Yep, updated mixer submodule
